### PR TITLE
Update vsts batch import

### PR DESCRIPTION
**Pull request by philipktlin**
Update to import pipeline task for a new perfix; and fix date time type parameter when calling to stored procedures; and all more logs.

### CODE REVIEW

File: tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs [hunk 1]
```diff
@@ -253,11 +264,11 @@ void UpsertVstsReleaseDeploymentToDb(SqlConnection sqlConnection, int releaseEnv
             cmd.Parameters.Add(new SqlParameter("@ReleaseEnvironmentId", releaseEnvironmentId));
             cmd.Parameters.Add(new SqlParameter("@Attempt", deployment.Attempt));
             cmd.Parameters.Add(new SqlParameter("@Status", deployment.Status.ToString()));
-            cmd.Parameters.Add(new SqlParameter("@LastModifiedOn", deployment.LastModifiedOn));
+            cmd.Parameters.Add(new SqlParameter("@LastModifiedOn", SqlDbType.DateTime2) { Value = deployment.LastModifiedOn });
```
Comment (and-rewsmith) [inline tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs]: Any reason why you needed to change the type?

File: tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs [hunk 2]
```diff
@@ -180,13 +182,22 @@ async Task ImportVstsReleasesDataAsync(ReleaseManagement releaseManagement, stri
                             {
                                 UpsertVstsReleaseDeploymentToDb(sqlConnection, releaseEnvironment.Id, deployment);
 
-                                foreach(IoTEdgePipelineTask pipelineTask in deployment.Tasks.Where(x => IsTestTask(x)))
+                                const string testTaskPrefix = "Test:";
```
Comment (and-rewsmith) [inline tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs]: My understanding is that you are adding a prefix to all test tasks in the test yaml then stripping the substring out when loading the tasks into the db. 

Do we have the changes in master yet to change the task names, or is it a hardcoded yaml in the pipeline?

File: tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs (hunk 1 above)
Comment (philipktlin) [reply tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs]: Previously the type is wrong, it is the right type to use; check what is done for vsts build.

File: tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs (hunk 2 above)
Comment (philipktlin) [reply tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs]: Old e2e tests is not in yaml; just in vsts pipeline. :(

**Pull request merged**